src/main/steps/applicant1/hub-page/sole/content.ts

```diff
@@ -460,6 +460,11 @@ const cy: typeof en = (
       : `${
           userCase.state === State.AwaitingClarification ? 'Beth sydd angen i chi ei wneud' : 'Diweddariad diweddaraf'
         }`,
+  finalOrderComplete: {
+    line1: 'Your final order has been granted. You are now legally divorced.',
+    line2: `<a class="govuk-link" href="/downloads/final-order-granted" download="Final-Order-Granted">Download a copy of your ‘final order’</a>.
```

Comment:
same here.

Reply:
Moved the links to nunjucks files and renamed references